packages/core/src/components/forms/controls.tsx

```diff
@@ -62,6 +79,7 @@ export class Control<P extends IControlProps> extends React.Component<React.HTML
                 <input {...inputProps} ref={inputRef} type={type} />
                 <span className={Classes.CONTROL_INDICATOR} />
                 {this.props.label}
+                {this.props.labelElement}
```

Comment:
I just can't imagine a case where setting those two props at the same time would be intentional. Probably a false concern 🤷‍♂️